logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -28,9 +28,9 @@
     public static final String VERSION = "@version";
     public static final String VERSION_ONE = "1";
 
-    private static final ObjectMapper mapper = new ObjectMapper();
-
-    // TODO: add metadata support
+    private static final Logger DEFAULT_LOGGER = new StdioLogger();
+    private transient final ObjectMapper mapper = new ObjectMapper();
+    private transient static Logger logger = DEFAULT_LOGGER;
```

Comment:
Should we use a singleton or factory pattern here instead of a global variable? Feels weird to have a mutable global variable.